### ISSUE TITLE
"signatureLocal: Cannot resolve signature;2"

### ISSUE THREAD

**Issue by BeastMode2018** [inbasic/iaextractor#215]
Since the latest FireFox update from Monday of this week, I can no longer download YouTube videos with YouTube Video and Audio Downloader. I push the integrated "Download" button and the program says "signatureLocal: Cannot resolve signature;2" 
Not sure if it's an issue with FireFox, YouTube of something else.

**Comment by RiVeND** [comment]
Yes, I've hit this as well. However, it's not all YouTube videos - just some of them. One that causes the error is:
https://www.youtube.com/watch?v=a_426RiwST8
One that doesn't:
https://www.youtube.com/watch?v=SbAMOQcNtAc

**Comment by BeastMode2018** [comment]
that's odd, but thanks for the info. 

----- Original Message -----
